apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex

```diff
@@ -170,6 +174,7 @@ defmodule OMG.WatcherInfo.DB.Block do
       |> prepare_inserts(db_txs_stream, "db_txs_", DB.Transaction)
       |> prepare_inserts(db_outputs_stream, "db_outputs_", DB.TxOutput)
       |> DB.TxOutput.spend_utxos(db_inputs)
+      |> Ecto.Multi.update("pending_block", PendingBlock.done_changeset(pending_block), [])
```

Comment:
why are you not dropping it?

Reply:
I was wondering if we should keep them as `done` or just delete them, wdyt?

Reply:
Dropping now